src/wagtail_personalisation/wagtail_hooks.py

```diff
@@ -208,9 +208,19 @@ def get_context(self):
             if root_page.is_root():
                 page_count -= 1
 
-            context["total_pages"] = page_count
+            return page_count
 
-        return context
+    if WAGTAIL_VERSION < (2, 15):
```

Comment:
Wagtail itself prefers the `>=` check
https://docs.wagtail.io/en/stable/contributing/python_guidelines.html#django-compatibility

suggestion:

```python
if WAGTAIL_VERSION >= (2, 15):
    def get_context_data(self, parent_context):
        # ...
else:
    def get_context(self):
        # ..

Reply:
Thanks @zerolab - changed as suggested